### ISSUE TITLE
Sharpness Bar View

### ISSUE THREAD

**Issue by neftalimich** [TurkeyTickle/mhw-builder#55]
Hi, I have been working on that. But I need your approval :)

- Weapons.tsv - example: 
Wyvern Ignition "Impact" 25;2;2;2;3;6 (all values must have a full bar data)

- Calculation.Service
![mhw-sharpness-bar](https://user-images.githubusercontent.com/10821661/42852033-6024a9d2-89f4-11e8-8ee1-4dcc9dcc17a3.png)

- View
![mhw-sharpness-bar-view](https://user-images.githubusercontent.com/10821661/42852041-67f1bf1a-89f4-11e8-94a4-efbb55cf6294.png)

- Database: I can get some info from mhw-db.com (but it does not have all the information)

**Comment by TurkeyTickle** [comment]
Very cool! Visually I think it looks great, but I think the code needs to be done differently. I'm concerned that manually injecting html with _ngcontent attributes will impact component encapsulation in unexpected ways. So instead of building the html in the calculation service, I would create a sharpness bar component, pass the sharpness data to it, then let the component template build the html by interpreting the sharpness data.

**Comment by neftalimich** [comment]
I get it. Thanks for everything, I'm learning a lot from you. What do you think about create a Discord chat to exchange ideas? I have a lot of improvements, but I have made a mess with my project and my commits. x)

**Comment by neftalimich** [comment]
Well, I don't know what I was thinking when I did such a thing.
how looks now? it's better? 

![captura](https://user-images.githubusercontent.com/10821661/42951773-928bb624-8b3c-11e8-9649-b5a1e63a04e4.PNG)

![captura](https://user-images.githubusercontent.com/10821661/42951978-14691358-8b3d-11e8-8a27-2a2a9fa16a24.PNG)

**Comment by TurkeyTickle** [comment]
Yeah, looks much better, nice job! I think getting a Discord set up is a good idea. I'll set one up soon and send you an invite.

**Comment by TurkeyTickle** [comment]
Here's the discord invite. I'll probably add a link to it either on the site itself or in the readme so other people can join as well.

https://discord.gg/QNpsupq
